### ISSUE TITLE
Add a "Lite" version that enables less features

### ISSUE THREAD

**Issue by Geovanne5** [IverCoder/a03-additional-features#1]
if you can... leave the default app opening animations and just leave the screen recorder

**Comment by IverCoder** [comment]
Can you share a good use case for a lite version? Does your phone currently face issues while using this module?

I'll prepare a version with those two modifications excluded when I get the time, but please do tell me any issues you face with the current version of the module.

**Comment by Geovanne5** [comment]
No, I'm not facing any problems but I wanted a version with the default app opening animations and with the blur effect reduced (if that's possible)
